src/Pages/Profile/admin/AdminView.jsx

```diff
@@ -38,6 +38,7 @@ export default function Editor(props) {
       lastName: lastName || user.lastName,
       middleInitial: middleInitial || user.middleInitial,
       email: email || queryEmail,
```

Comment:
`email` should not be updated on the backend. this causes a logistic headache. change the API to send only `email` as a parameter which will find the user in a search.

Reply:
let make an issue for this. Its an easy fix and i don't think it should be here

Reply:
my latest push dealt with this